eos/form-factors/observables.cc

```diff
@@ -1616,6 +1645,7 @@ namespace eos
 
                 // B -> V
                 make_b_to_omega_form_factors_group(),
+                make_b_to_gammastar_form_factors_group(),
```

Comment:
I don't think that ``PToGammaOffshell`` should be listed under ``B -> V``. Please put it into a category of its own, and coordinate with @pluegh to join the ``B->gamma`` and ``B->gamma^*`` form factors.